mock/mock.go

```diff
@@ -52,6 +52,8 @@ type Call struct {
 	// receives a message or is closed. nil means it returns immediately.
 	WaitFor <-chan time.Time
 
+	WaitTime time.Duration
```

Comment:
Unexport this, so it has to be set with `After`.

Reply:
Followed thinking its a convention, because other members are exported.